assets/javascript/javascript.js

```diff
@@ -0,0 +1,117 @@
+// Global Variables
+var drkAether;
+var drgAether;
+var ninAether;
+var samAether;
+var goalAether;
+var totalAether;
+var win = 0;
+var loseAlert = ["Who needs a lost count when you're a winner at heart",
```

Comment:
Alerts stop the code so it does not fully finish displaying the numbers.  I would check into a modal to pop up the result with a "reset" or "play again" button.